lexicons/tools/ozone/moderation/emitEvent.json

```diff
@@ -44,9 +44,19 @@
             },
             "subjectBlobCids": {
               "type": "array",
-              "items": { "type": "string", "format": "cid" }
+              "items": {
+                "type": "string",
+                "format": "cid"
+              }
             },
-            "createdBy": { "type": "string", "format": "did" }
+            "createdBy": {
+              "type": "string",
+              "format": "did"
+            },
+            "userAgent": {
+              "type": "ref",
+              "ref": "tools.ozone.moderation.defs#userAgent"
+            }
```

Comment:
My only confusion about this kind of usage is that there is also the `user-agent` header which will be present in these requests.